### PR TITLE
Leave systemd unit

### PR DESCRIPTION
**Pull request by elendrys**
We do not need to overwrite the unit file.

### CODE REVIEW

**Comment by Aversiste** [comment]
Can you drop commit f15e74b from this PR please?

File: tasks/install/Debian.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 - name: Debian - Ensure haproxy apt repo is added.
   apt_repository:
-    repo: "deb {{ haproxy_repo }}"
+    repo: "deb {{ haproxy_repo }} {{ ansible_distribution_release }} main"
```
Comment (Aversiste) [inline tasks/install/Debian.yml]: This is a revert of f15e74b. Is this really what you want to do?

File: vars/Debian.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-_haproxy_repo: "http://http.debian.net/debian {{ ansible_distribution_release }} main"
+_haproxy_repo: "http://http.debian.net/debian"
```
Comment (Aversiste) [inline vars/Debian.yml]: Same here.

File: tasks/install/Debian.yml (hunk 1 above)
Comment (elendrys) [reply tasks/install/Debian.yml]: Sorry I didn't apply the rebase correctly in this branch

File: vars/Debian.yml (hunk 2 above)
Comment (elendrys) [reply vars/Debian.yml]: Sorry I didn't apply the rebase correctly in this branch

File: tasks/install.yml [hunk 3]
```diff
@@ -11,6 +11,11 @@
     - "install/{{ ansible_distribution }}.yml"
     - "install/{{ ansible_os_family }}.yml"
 
+- name: Check if the systemd unit is installed by the package
+  stat:
+    path: /lib/systemd/system/haproxy.service
+  register: haproxySvcUnit
```
Comment (olo-dw) [inline tasks/install.yml]: We are facing a consistency issue. This is a camelCase typography and with Ansible we are used to use snake_case typography.

Could you please be so kind to convert this register variable?

**Pull request merged**